liquibase-standard/src/main/java/liquibase/analytics/LiquibaseAnalyticsListener.java

```diff
@@ -123,7 +123,9 @@ private synchronized void flush() throws Exception {
                         logger,
                         logLevel,
                         "Sending anonymous data to Liquibase analytics endpoint. ",
-                        "Response from Liquibase analytics endpoint: ");
+                        "Response from Liquibase analytics endpoint: ",
+                        0,
```

Comment:
Should we use the parameter here?

Reply:
No, that will occur here: https://datical.atlassian.net/browse/DAT-19744